docs/concepts.rst

```diff
@@ -369,8 +369,7 @@ for some basic load balancing and fault tolerance when used in conjunction
 with retries.
 
 Airflow also has the ability to reference connections via environment
-variables from the operating system. But it only supports URI format. If you
-need to specify ``extra`` for your connection, please use web UI.
```

Comment:
Please include how to specify the extra params - as I didn't even know this was possible!

Reply:
I'm glad that I could show you something new in the project :-) 
Tomorrow I will introduce an explanation.